core/src/databases.rs

```diff
@@ -6,7 +6,7 @@ use bson::Bson;
 #[derive(Debug)]
 pub struct MongoDatabases {
     // The list of all the databases
-    databases_names: Vec<String>,
+    database_names: Vec<Bson>,
```

Comment:
I changed this from `String` to `Bson` as I was having trouble with `get_value()` and the return Bson value existing after the function return as that returns a reference to a `Bson`.  Let me know if there is a better way or if this is okay.

Reply:
I made the change in get_value() to return a `Bson` rather than the reference `&Bson`, which is what we discussed.  With that I am able to revert back to a `Vec<String>` in the MongoDatabases struct and avoid the extra loop of converting the vector of String to Bson.  
Thanks for sharing the info on the impact of iterating through the list of DBs to convert it to a Bson.